src/main/java/com/gmail/yuramitryahin/controller/RegistrationManufacturerController.java

```diff
@@ -0,0 +1,31 @@
+package com.gmail.yuramitryahin.controller;
+
+import com.gmail.yuramitryahin.lib.Injector;
+import com.gmail.yuramitryahin.modal.Manufacturer;
+import com.gmail.yuramitryahin.service.ManufacturerService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class RegistrationManufacturerController extends HttpServlet {
```

Comment:
```suggestion
public class CreateManufacturerController extends HttpServlet {
```
controller name should starts with the verb, let's change it in each of them